lib/live_view/state/element_key.dart

```diff
@@ -0,0 +1,28 @@
+class ElementKey {
+  ElementKey(this.key, [this.component]);
+
+  final String key;
+  final String? component;
+  get isComponent => component != null;
+
+  @override
+  bool operator ==(Object other) {
+    if (other is! ElementKey) return false;
+    if (key != other.key) return false;
+    if (component != other.component) return false;
+    return true;
+  }
+
+  @override
+  int get hashCode {
+    var result = 17;
+    result = 37 * result + key.hashCode;
+    result = 37 * result + component.hashCode;
+    return result;
```

Comment:
```suggestion
    return hash2(key, component);
```

I think usually you would use hash2 for that

Reply:
Thanks